Description: Don't run make -C kernel clean
 The kernel modules are no longer built (part of mainline), so don't
 run make -C kernel clean on make clean.
Author: Christian Seiler <christian@iwakd.de>
Forwarded: not-yet
Last-Update: 2016-07-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -75,7 +75,6 @@ clean:
 	$(MAKE) -C utils/fwparam_ibft clean
 	$(MAKE) -C utils clean
 	$(MAKE) -C usr clean
-	$(MAKE) -C kernel clean
 	[ ! -f iscsiuio/Makefile ] || $(MAKE) -C iscsiuio clean
 	[ ! -f iscsiuio/Makefile ] || $(MAKE) -C iscsiuio distclean
 
